src/models/Tweet.ts

```diff
@@ -0,0 +1,48 @@
+import { ObjectId } from "mongodb";
```

Comment:
rename file to Tweet.model.ts

Reply:
sorry, remember to do it, but appeared to not.